### DISCUSSION TITLE
mrf file naming convention for different locales

### DISCUSSION THREAD

**Discussion by phamk1** [CMSgov/hospital-price-transparency#129] [Q&A] [answered]
Hi,

We have separate files for rates that are determined by locale.  How do you advise we name the file to distinguish between the locales?  The MRF naming wizard allows you to add an optional location by NPI.  However, our locations share the same NPI but are in different locales with different rates.  ie NYC, NY Suburbs.

Thanks in advance.

**Comment by RidenourK** [comment, marked as answer]
@phamk1. Thank you for your question.

If a hospital has multiple facilities/locations, each with a different set of standard charges, functioning under the same EIN, legal name, and NPI, the hospital may add additional information in the MRF name after the hospital’s legal name to further distinguish the location. Example: example-hospital-east-campus.
